data/deprecated.json

```diff
@@ -2031,5 +2031,21 @@
   {
     "old": {"industrial": "brickworks"},
     "replace": {"industrial": "brickyard"}
+  },
+  {
+     "old": {"cycleway": "opposite"},
+     "replace": {"oneway:bicycle": "no", "cycleway:both": "no"}
+  },
+  {
+     "old": {"cycleway:left": "opposite"},
+     "replace": {"oneway:bicycle": "no", "cycleway:left": "no"}
+  },
+  {
+     "old": {"cycleway": "opposite_lane"},
+     "replace": {"oneway:bicycle": "no", "cycleway:left": "lane", "cycleway:left:oneway": "-1"}
```

Comment:
I think this rule is wrong in countries that drive on the left (which affects [67 ways](https://overpass-turbo.eu/s/1OWv))

It's also wrong if the road has `oneway=-1` (which affects [2 ways](https://overpass-turbo.eu/s/1OWw))

So I think these 69 examples need to be manually fixed before we can introduce the rule

...or we could make a fairly simple change to iD, so that deprecation rules can be limited to certain countries or driving sides. (we already have this data from https://github.com/rapideditor/country-coder)

Reply:
Thanks for confirming the issue on left-hand traffic!

I will wait for https://github.com/openstreetmap/id-tagging-schema/pull/1295 to be merged before thinking more about  this, though. Just not to invest too much time – given our longer backlog. :)